app-route.html

```diff
@@ -236,9 +237,8 @@
 
       __resetProperties: function() {
```

Comment:
Heya, Tim, spoke with Chris on the issue in person about this. I thought that it was a specific use case he was gunning for, but I was mistaken. There is no need to implement that part of my suggestion.

Although, I still believe that `__resetPeoperties` should be renamed to `__deactivate` because that seems to be it's main purpose now rather than resetting `data`, `tail`, and `__matched` as it did in it's first implementation.

Thoughts?